### DISCUSSION TITLE
Can we run lighthouse multiple times on the same commit hash?

### DISCUSSION THREAD

**Discussion by chatur-sriganesh** [GoogleChrome/lighthouse-ci#733] [Q&A]
Currently when i run lhci autorun for the second time i get an error "Build already exists for hash \"<hash code>\"

Use case: It would be greatly helpful for me to view the performance trend of my application for every code change i make locally, for the same commit.
@patrickhulce  any help/ideas in this regard please?

Thanks,
Chatur

**Comment by patrickhulce** [comment]
I strongly recommend that, even locally, you make different commits at each checkpoint that you wish to measure. This will make it much easier to understand what change had what impact and you can always rebase/squash your commits at the end (not to mention it fits the *exact* use case of LHCI).
